client/app/templates/components/game-client.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span class="glyphicon glyphicon-comment"></span> Client
+        </div>
+        <div class="panel-body">
+          <ul class="chat">
+            {{#each messages as |msg|}}
+              <li class="{{if msg.isServer "left" "right"}} clearfix">
```

Comment:
I really don't like the styling. It's just awkward with things swapped. We can deal with it later but I would want it more consistent with more of a subtle difference between what you typed and what the server returned.